dev-haskell/amazonka-cloudsearch-domains/files/amazonka-cloudsearch-domains-2.0-duplicate-record-fields.patch

```diff
@@ -0,0 +1,330 @@
++{-# LANGUAGE NoImplicitPrelude     #-}
++{-# LANGUAGE OverloadedStrings     #-}
++{-# LANGUAGE StrictData            #-}
+ {-# OPTIONS_GHC -fno-warn-unused-matches #-}
+ 
+ -- Derived from AWS service descriptions, licensed under Apache 2.0.
+-
+ -- |
+ -- Module      : Amazonka.CloudSearchDomains.Types
+ -- Copyright   : (c) 2013-2023 Brendan Hay
+@@ -15,190 +14,148 @@
+ -- Portability : non-portable (GHC extensions)
+ module Amazonka.CloudSearchDomains.Types
+   ( -- * Service Configuration
+-    defaultService,
+-
+-    -- * Errors
+-    _DocumentServiceException,
+-    _SearchException,
+-
+-    -- * ContentType
+-    ContentType (..),
+-
+-    -- * QueryParser
+-    QueryParser (..),
+-
+-    -- * Bucket
+-    Bucket (..),
+-    newBucket,
+-    bucket_count,
+-    bucket_value,
+-
+-    -- * BucketInfo
+-    BucketInfo (..),
+-    newBucketInfo,
+-    bucketInfo_buckets,
+-
+-    -- * DocumentServiceWarning
+-    DocumentServiceWarning (..),
+-    newDocumentServiceWarning,
+-    documentServiceWarning_message,
+-
+-    -- * FieldStats
+-    FieldStats (..),
+-    newFieldStats,
+-    fieldStats_count,
+-    fieldStats_max,
+-    fieldStats_mean,
+-    fieldStats_min,
+-    fieldStats_missing,
+-    fieldStats_stddev,
+-    fieldStats_sum,
+-    fieldStats_sumOfSquares,
+-
+-    -- * Hit
+-    Hit (..),
+-    newHit,
+-    hit_exprs,
+-    hit_fields,
+-    hit_highlights,
+-    hit_id,
+-
+-    -- * Hits
+-    Hits (..),
+-    newHits,
+-    hits_cursor,
+-    hits_found,
+-    hits_hit,
+-    hits_start,
+-
+-    -- * SearchStatus
+-    SearchStatus (..),
+-    newSearchStatus,
+-    searchStatus_rid,
+-    searchStatus_timems,
+-
+-    -- * SuggestModel
+-    SuggestModel (..),
+-    newSuggestModel,
+-    suggestModel_found,
+-    suggestModel_query,
+-    suggestModel_suggestions,
+-
+-    -- * SuggestStatus
+-    SuggestStatus (..),
+-    newSuggestStatus,
+-    suggestStatus_rid,
+-    suggestStatus_timems,
+-
+-    -- * SuggestionMatch
+-    SuggestionMatch (..),
+-    newSuggestionMatch,
+-    suggestionMatch_id,
+-    suggestionMatch_score,
+-    suggestionMatch_suggestion,
+-  )
+-where
+-
+-import Amazonka.CloudSearchDomains.Types.Bucket
+-import Amazonka.CloudSearchDomains.Types.BucketInfo
+-import Amazonka.CloudSearchDomains.Types.ContentType
+-import Amazonka.CloudSearchDomains.Types.DocumentServiceWarning
+-import Amazonka.CloudSearchDomains.Types.FieldStats
+-import Amazonka.CloudSearchDomains.Types.Hit
+-import Amazonka.CloudSearchDomains.Types.Hits
+-import Amazonka.CloudSearchDomains.Types.QueryParser
+-import Amazonka.CloudSearchDomains.Types.SearchStatus
+-import Amazonka.CloudSearchDomains.Types.SuggestModel
+-import Amazonka.CloudSearchDomains.Types.SuggestStatus
+-import Amazonka.CloudSearchDomains.Types.SuggestionMatch
+-import qualified Amazonka.Core as Core
+-import qualified Amazonka.Core.Lens.Internal as Lens
+-import qualified Amazonka.Prelude as Prelude
+-import qualified Amazonka.Sign.V4 as Sign
```

Comment:
In the future, please be cautious not to make formatting changes like this, since they add nothing except extra bytes to the patch. Something I'll often do is make a throwaway git repo in the source directory so that it's easy to look at changes before creating the patch.

e.g.
```bash
pn=amazonka-cloudsearch-domains

cd /tmp
cabal get $pn && cd $pn-*/
git init && git add . && git commit -m init

vi gen/Amazonka/CloudSearchDomains/Types.hs
git diff # double check there aren't unneeded changes in the diff

git add .
git commit -s -m 'Fix gen/Amazonka/CloudSearchDomains/Types.hs'
git format-patch -n1 --stdout > $(get_repo_path / haskell)/$pn/files/$pn-2.0-duplicate-record-fields.patch
```